New_PlatformGame/Assets/NewPlayerMovement.cs

```diff
@@ -53,23 +44,37 @@ void Update()
         {
             Jump();
         }
-        SetJumpBar(_jumpBar, _jumpBarSprite);
+        SetJumpBar();
     }
 
-    private void SetJumpBar(GameObject jumpBar, SpriteRenderer jumpBarSprite) //점프 바 설정
+    private void SetJumpBar() //점프 바 설정
     {
+        Vector3 v = Camera.main.ScreenToWorldPoint(new Vector3(Screen.width, Screen.height, 0));
```

Comment:
```suggestion
        Vector3 v = Camera.main.ScreenToWorldPoint(new Vector3(Screen.width, Screen.height, 0));
```

`new Vector3(Screen.width, Screen.height, 0)` 이 값은 항상 고정되어 있는 값이니까, 다음과 같이 const나 static으로 상수 혹은 하나만 있을 수 있는 객체로 만들어주면 리소스를 덜 잡아먹는다. new Vector3를 쓰게되면 이 메소드를 부를 때마다 새로운 Vector3 객체를 하나씩 만들게 되니까, 가능한 이렇게 똑같이 재활용 하는 변수는 상수로 돌리는게 좋다.

```cs
const Vector3 camera = new Vector3(Screen.width, Screen.height, 0);
```
이렇게 만들어놓고 `Vector3 v = Camera.main.ScreenToWorldPoint(camera)` 이렇게만 써도 훨씬 나은 퍼포먼스를 기대할 수 있다. ScreenToWorldPoint는 카메라 값을 현재 월드상으로 바꿔주는 거니까 상수로 만들면 안되고.